changelog/release-schedule.dd

```diff
@@ -0,0 +1,56 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(UL
+    $(LI New release are published every $(I two) months, on the first day of every uneven month.)
+    $(LI Two weeks before a new release `master` is merged into `stable` and a first beta is released.)
+    $(LI Patch releases are usually released on the first day of every even month.)
```

Comment:
Patch releases are still unplanned. There is a good argument for releasing fixes asap. Sometimes we just don't have enough important fixes to warrant the effort of a patch release.